lab_2_tokenize_by_bpe/main.py

```diff
@@ -69,6 +130,20 @@ def get_vocabulary(
     :param unknown_token: a token to signify an unknown token
     :return: dictionary in the form of <token: identifier>
     """
+    if not isinstance(word_frequencies, dict) or\
+            not isinstance(unknown_token, str):
+        return None
+    list_of_words = []
+    get_vocabulary_dict = {}
+    for word in word_frequencies.keys():
+        for token in str(word):
```

Comment:
This way you are splitting the word into unigrams again...

Reply:
Fixed

Reply:
As I understand I should go through tuples, words and symbols there and add words and symbols to the list